src/Attachment.tsx

```diff
@@ -170,34 +169,44 @@ export const AttachmentView = (props: {
         case "application/vnd.microsoft.card.hero":
             if (!attachment.content)
                 return null;
+
+            const heroCardBuilder = new CardBuilder.AdaptiveCardBuilder();
```

Comment:
Or, maybe create derived classes from AdatpiveCardBuilder for the types? I'm not sure which would look better.